src/object/srv_ec_aggregate.c

```diff
@@ -1227,7 +1227,7 @@ agg_peer_update_ult(void *arg)
 			    DAOS_OBJ_RPC_EC_AGGREGATE, &rpc);
 	if (rc) {
 		D_ERROR("obj_req_create failed: "DF_RC"\n", DP_RC(rc));
-		goto out;
+		goto out_rpc;
```

Comment:
this seems wrong.   The rpc should only be set if obj_req_create succeeds.   In this case, it seems coverity would be complaining that rpc is not NULL initialized.   In fact, this can make it worse because rpc could be garbage and therefore it will call crt_req_decref even if it fails.

Reply:
@jolivier23  ok, I will update to just return then and remove the out_rpc goto. Does it still need to set the eventual in the case that obj_req_create fails? or can it just return?

Reply:
I think my error was to not initialize the crt_rpc_t * to NULL.  If obj_req_create fails my handling in the goto out: tests for NULL and then calls crt_req_decref(rpc) on this pointer in not NULL.  I'm not sure what Coverity is complaining about, but this seems to be something that should be done (initial rpc pointer to NULL).

The eventual should be set before exiting the function.